src/mqueryfront/src/QueryField.js

```diff
@@ -164,7 +165,7 @@ class QueryField extends Component {
                             name="parse"
                             type="submit"
                             onClick={(event) =>
-                                this.handleQuery(event, "parse")
+                                this.handleQuery(event, "parse", "parse")
```

Comment:
:thinking: this task doesn't really have (any) priority, so why it is required?

Reply:
Hmm, long story. Yeah, not required in this case (TLDR earlier priority was in the path and `query/sth/undefined` looked ugly